pkg/btf/validation.go

```diff
@@ -53,6 +55,15 @@ func ValidateKprobeSpec(bspec *btf.Spec, call string, kspec *v1alpha1.KProbeSpec
 		if err == nil {
 			err = bspec.TypeByName(call, &fn)
 		}
+		if errors.Is(err, btf.ErrMultipleMatches) {
+			var types []btf.Type
```

Comment:
could you please add some warning that it's happening?

Reply:
Yes makes sense, will add it and will also make it work on non syscall, since I have some kprobes hook tests here, will update soon, thank you!

Reply:
done ;-)